lib/tape/installer.rb

```diff
@@ -4,7 +4,7 @@ class Installer < ExecutionModule
 
     action :install,
       proc {install},
-      'Creates all nessisary hosts and config files'
+      'Creates all necessary hosts and config files'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.